package/gluon-wlan-encryption-psk/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon wlan encrypt
+Encrypt wireless networks in a non-Freifunk setup.
+
+This package allows to encrypt 802.11s and infrastructure networks using a pre shared key (psk).
+
+This setup results in passwords being stored in plain-text. Site-configuration, Firmware-downloads and
+Devices must be protected against unauthorized access
+
+*make sure to use exclude hostapd-mini site.mk*
```

Comment:
Can you please explain this better?

Reply:
What I would wish to have here is a simple explanation about secure WPA passwords like:

Please be aware that the shortest allowed PSK's length is 8 characters and it can be up to 63 characters long. It is highly recommended to choose a randomly generated PSK with at least 40 characters consisting of alphanumeric, numeric and special characters.

Reply:
What I would wish to have here is a simple explanation about secure WPA passwords like:

Please be aware that the shortest allowed PSK's length is 8 characters and it can be up to 63 characters long. It is highly recommended to choose a randomly generated PSK with at least 40 characters consisting of alphanumeric, numeric and special characters.

Reply:
I do not agree with this. It is clear that a WPA password is to be used. The implications of password security are also clear. I do not see the need to elaborate as part of the gluon project.

Reply:
> Can you please explain this better?

could the package exclude hostapd-mini?